content/learn/book/welcome/installation/_index.md

```diff
@@ -95,6 +95,40 @@ Now run `cargo run` again. The Bevy dependencies should start building. This wil
 
 Now that we have our Bevy project set up, we're ready to start making our first Bevy app!
 
+### Hello Bevy World
+
+Within `main.rs`, let's create our first app!
+
+```rust
+use bevy::prelude::*;
+
+fn main(){
+  App::build().run();
+}
+```
+
+This won't report anything interesting: it merely creates an empty app with an empty **world**.
+
+Let's add our first system by modifying the file to the following:
+
+```rust
+use bevy::prelude::*;
+
+fn main(){
+  App::build()
+  .add_system(hello.system())
+  .run();
+}
+
+fn hello(){
+    println!("Hello, Bevy!")
+}
+```
+
+Systems in Bevy are just ordinary Rust functions, which are registered in our app to operate on our world.
+
+Calling `cargo run` will run our schedule once, causing `hello` to print "Hello, Bevy!", allowing us to verify our installation of Bevy.
```

Comment:
I think that the hello world example is better suited in the App section, where it is already fully quoted.

The installation guide is often partially or completely skipped by most users, since many are already familiar to the Rust ecosystem. Also, a basic app is not part of the installation.

Reply:
Hmm. I am uncomfortable ending the installation guide without a proper check that things "actually work". This isn't quite the right tool for that though; maybe it should use DefaultPlugins instead.

Reply:
Improved in b0a4951c7a6423045a37dab05738240f56265167